socorro/external/es/supersearch.py

```diff
@@ -505,12 +523,30 @@ def get(self, **kwargs):
                     hits = []
                     total = 0
                     aggregations = {}
+                    shards = None
                     break
 
+        if shards and shards.failed:
```

Comment:
It's an ``AttrDict`` returned by ``elasticsearch-dsl``. It's basically like our ``DotDict`` objects.